.ci/install_macos.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -x
+set -e
 # Add e option once we move to using Brewfile for installing packages.
```

Comment:
Can we remove this comment since this starts using `-e`?

Reply:
Right, the comment and option should be consistent.

More importantly, I'm still investigating whether this option causes the build failures on macOS (I'm testing it in `docker_v2_test` branch).